### PR TITLE
APPS-120 - Added an option to check for single approver

### PR DESCRIPTION
**Pull request by shiva-sc**

### CODE REVIEW

**Comment by shiva-sc** [comment]
@duboisp @jonasg-gc @yeungca2 Please review the PR

**Review by jonasg-gc — APPROVED** [review]
Looks good to me.

**Comment by jonasg-gc** [comment]
@duboisp : did you also want to review?

**Review by duboisp — CHANGES_REQUESTED** [review]

File: server.js [hunk 1]
```diff
@@ -231,8 +231,9 @@ MongoClient.connect( processEnv.MONGODB_URI || '', {useUnifiedTopology: true} ).
 	app.get('/api/v1/mailing/:mailingid/history',
 		userController.isAuthenticated,
 		mailingController.v_mailingHistory);
-	app.get('/api/v1/mailing/:mailingid/approval',
+	app.get('/api/v1/mailing/:mailingid/:subscode/approval',
 		userController.isAuthenticated,
+		bodyParser.urlencoded({extended:true, limit: '1024k'}),
```
Comment (duboisp) [inline server.js:236]: Do we need this?

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by duboisp — CHANGES_REQUESTED** [review]

File: controllers/mailing_view.js [hunk 2]
```diff
@@ -208,19 +208,20 @@ exports.v_mailingCancelled = async ( req, res, next ) => {
 	res.redirect( _baseRedirFolder + mailingId + "/edit" );
 }
 
-
 exports.v_mailingApproval = async ( req, res, next ) => {
 	// Send a test email to the predefined list of emails
 	// Set state to "completed"
 	
 	const mailingId = req.params.mailingid;
+	const subscode = req.params.subscode;
```
[the remark this answers is not in the document]
Comment (duboisp) [reply controllers/mailing_view.js:216]:
```suggestion
	const subscode = req.params.subscode || "0";
```

**Review by duboisp — APPROVED** [review]
Thanks Shiva

**Comment by duboisp** [comment]
For the record: This is a MINOR change.

\cc @jonasg-gc

**Pull request merged**